src/cljam/io/util/bin.clj

```diff
@@ -37,8 +40,18 @@
 (defn get-spans
   "Calculate span information for random access from index data such as tabix."
```

Comment:
```suggestion
   "Calculates span information for random access from index data such as tabix."
```
As cljam's convention, docstring uses third-person singular verbs. Please fix it if you notice old misspellings.